tests/protractor/utils.js

```diff
@@ -89,22 +92,27 @@ var revertSettingsForDdoc = function (ddocName) {
     path: path.join('/', constants.DB_NAME, '_design', mainDdocName, '_rewrite/update_settings', ddocName, '?replace=1'),
     method: 'PUT',
     body: JSON.stringify(originalSettings[ddocName])
-  }).then(function () {
+  }).then(function() {
     delete originalSettings[ddocName];
   });
 };
 
+const writeScreenShot=(data, filename)=> {
```

Comment:
After correcting all the whitespace in the other function declarations, you might make this one consistent with them.